### ISSUE TITLE
Mat-Slide-Toggle on Edge: The toggle switches only once by double-click

### ISSUE THREAD

**Issue by randomUser222** [angular/components#17490]
#### Reproduction

Reproducable on the documentation example:
https://material.angular.io/components/slide-toggle/examples

Steps to reproduce:
1. Double-click on the slide toggle
2. See that the toggle was switched only once
 
 
#### Expected Behavior

After double-clicking the slide-toggle, the value should be changed twice, same as it does on Chrome

#### Actual Behavior

After double-clicking the slide-toggle, the value is changed only once

#### Environment

- Angular: 8.0.2
- CDK/Material: 8.0.1
- Browser(s): Edge
- Operating System (e.g. Windows, macOS, Ubuntu): Windows

**Comment by devversion** [comment]
Thanks for the issue. I tried looking into this. It seems like this is an actual issue for native checkboxes in IE11 too. Since we rely on native `<label>` and `<input type="checkbox>` for the slide-toggle, the issue can be observed as well.

I don't think this is something we can fix easily because it would involve detecting changes differently  and would most likely destroy the concept of relying on native inputs for accessibility. It looks like this is a uncommon issue that only surfaces in IE11, so this is low priority.

**Comment by crisbeto** [comment]
Closing since the non-Chromium version of Edge and IE11 won't be supported as of version 13.
